### DISCUSSION TITLE
What is the correct setup for the Lian Li L-Connect-3 when using Fan Control?

### DISCUSSION THREAD

**Discussion by LionHeartBlue** [Rem0o/FanControl.Releases#1884] [Q&A] [answered]
I started using Fan Control recently and it is amazing, best software I have found for this purpose.  However, I don't know how I should correctly configure my system in order to take advantage of this.

I have 10 Lian Li Fans in connected in 4 different groups, via the Lian Li Hub.   Historically I was using the L Connect 3 software to control the fans and the ARGB lighting.

If I have L Connect running then it seems to take over the control and Fan Control has no effect.  If close L Connect then Fan Control takes over but the Fan Lights are no longer controllable and they remain off.

If I set L Connect to use Motherboard for Fan Speed and Lighting, then the fans seem to spin at full speed as if something else is interfering, and the Aromora Crate, Aura Sync doesn't seem to be able to control my lights.

So, have I plugged the fans in wrong?  Should I remove L Connect?  What is the correct process here, please? :D

Many thanks

**Comment by Rem0o** [comment, marked as answer]
I'm guessing you are using the FanControl.LianLi plugin?

If so, have you seen that information ? https://github.com/EightB1ts/FanControl.LianLi#known-issues

**Reply by LionHeartBlue** [reply]
After looking at the above tutorial again, I thought I would double-check that aII was plugged into the correct ARGB ports on the MB.  Upon doing so I noticed the cable had fallen out of the LIAN LI Controller, so the Fans could only be controlled via the L-Connect Software as the Controler was powered by USB.  Once plugged in correctly it all worked.
